cmd/manager/kodata/serving_v1alpha1_knativeserving_crd.yaml

```diff
@@ -0,0 +1 @@
+serving_v1alpha1_knativeserving_crd.yaml
```

Comment:
This is a broken link. You need to link the same way as knative-serving-0.6.0.yaml, like
../../../config/crds/serving_v1alpha1_knativeserving_cr.dyaml.